src/Kovan-DssSpell.sol

```diff
@@ -16,39 +16,273 @@
+            outputConduitID: "RWA004_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA004",
+            ilk: "RWA004-A",
+            ilkRegistryName: "RWA004-A: Centrifuge: Harbor Trade Credit",
+            RATE: SEVEN_PCT,
+            CEIL: 7 * MILLION,
+            PRICE: 8_014_300 * WAD,
+            MAT: 11_000,
+            TAU: 0,
+            DOC: ""
+        });
+
+        CentrifugeCollateralValues memory RWA005 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x1233d0DBb55A4Bb41D711d4B584f8DDB15A2Ff88,
+            GEM: 0xcB2A48D26970eE7193d66BAc6F1b3090f2E8f82B,
+            OPERATOR: 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319,
+            INPUT_CONDUIT: 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319,
+            OUTPUT_CONDUIT: 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319,
+            URN:  0xdB9f0700EbBac596CCeF5b14D5e23664Db2A184f,
+            LIQ: 0x2881c5dF65A8D81e38f7636122aFb456514804CC,
+            gemID: "RWA005",
+            joinID: "MCD_JOIN_RWA005_A",
+            urnID: "RWA005_A_URN",
+            inputConduitID: "RWA005_A_INPUT_CONDUIT",
+            outputConduitID: "RWA005_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA005",
+            ilk: "RWA005-A",
+            ilkRegistryName: "RWA005-A: Centrifuge: Fortunafi",
+            RATE: FOUR_PT_FIVE_PCT,
+            CEIL: 15 * MILLION,
+            PRICE: 16_380_375 * WAD,
+            MAT: 10_500,
+            TAU: 0,
+            DOC: ""
+        });
+
+        CentrifugeCollateralValues memory RWA006 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x039B74bD0Adc35046B67E88509900D41b9D95430,
+            GEM: 0x4E65F06574F1630B4fF756C898Fe02f276D53E86,
+            OPERATOR: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            INPUT_CONDUIT: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            OUTPUT_CONDUIT: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            URN:  0x6fa6F9C11f5F129f6ECA4B391D9d32038A9666cD,
+            LIQ: 0x2881c5dF65A8D81e38f7636122aFb456514804CC,
+            gemID: "RWA006",
+            joinID: "MCD_JOIN_RWA006_A",
+            urnID: "RWA006_A_URN",
+            inputConduitID: "RWA006_A_INPUT_CONDUIT",
+            outputConduitID: "RWA006_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA006",
+            ilk: "RWA006-A",
+            ilkRegistryName: "RWA006-A: Centrifuge: Peoples Company",
+            RATE: TWO_PCT,
+            CEIL: 20 * MILLION,
+            PRICE: 20_808_000 * WAD,
+            MAT: 10_000,
+            TAU: 0,
+            DOC: ""
+        });
+
+        CentrifugeCollateralValues[4] memory collaterals = [RWA003, RWA004, RWA005, RWA006];
+
+        // integrate RWA003-006
+        for (uint i = 0; i < collaterals.length; i++) {
+            integrateCentrifugeCollateral(collaterals[i]);
+        }
+
+        // increase debt ceiling of RWA002 from 5M to 20M
+        DssExecLib.increaseGlobalDebtCeiling(15 * MILLION);
+        DssExecLib.setIlkDebtCeiling("RWA002-A", 20 * MILLION);
```

Comment:
Do we also need to `bump` the price in Kovan?

Reply:
yes

Reply:
Just added this, as well as a generalized version of the RWA values test you created for the mainnet spell.